### ISSUE TITLE
Adding links to documentation

### ISSUE THREAD

**Issue by harrisonturton** [AnomalyInnovations/serverless-stack-com#227]
Hello,

Had a great time reading through this guide, but I think it'd really benefit from relevant links to the documentation.

When trying to strike out on my own (after going through this), I found it difficult to go past what the tutorial offers, because I didn't know where to go. The AWS docs are confusing for a beginner!

**Comment by jayair** [comment]
@harrisonturton Yeah they are. Can you give me an example of some links you'd like to see?

**Comment by harrisonturton** [comment]
These are a few things off the top of my head:

- Docs for working with DynamoDB, or at least a starting point
    - The tutorial explains how to Create/Add/Delete etc, but (obviously) Dynamo can do much more. Where do I go to find these?
    - Linking to the docs for just creating/adding/deleting items is probably sufficient, as the reader can discover more from there.
- Docs for the serverless framework
    - E.g. different options for the `serverless.yaml` file
    - I tried to follow along using a different language (Go) and got stuck due to differences in the framework. Links to the JS docs would've helped me find the relevant pages in the Go documentation.

**Comment by jayair** [comment]
I see. Yeah we should expand on the DynamoDB part for sure. I think this is a great guide - https://www.dynamodbguide.com.
